### ISSUE TITLE
Shouldn't it be :/var/lib/postgresql/data/

### ISSUE THREAD

**Issue by ls-saadabbasi** [itsdevcoffee/chainlink-fun#3]
Current the mapped volume seems like `<on-your-host>:/var/lib/postgressql/data/`
I'm not sure it should really be **postgreSSql** rather than **postgresql**.

ANYWAYS, 
soon as i change it to `:/var/lib/postgresql/data/` it starts persistence but chainlink_node stops working with error saying
`Unable to initialize ORM: database is newer than current chainlink version` 

If you also face this problem please just remove the volume you're using for you database persistence and try again

**Comment by entmike** [comment]
You are right.  It should be `/var/lib/postgresql/data/` and not `/var/lib/postgressql/data/`.  I just wasted 30 minutes figuring this out, too :)  My data kept disappearing when restarting my container as a result until I figured this out.
